.github/workflows/build_and_test.yaml

```diff
@@ -64,9 +64,9 @@ jobs:
     - name: Build and Push EG Commit Image
       if: github.event_name == 'push'
       # tag is set to the short SHA of the commit
-      run: make image.push.multiarch PLATFORMS="linux_amd64 linux_arm64"
+      run: make image.push.multiarch PLATFORMS="linux_amd64 linux_arm64" IMAGE=envoyproxy/gateway-dev
```

Comment:
is this change needed or just added it here to make it more explicit ?

Reply:
Not needed, but to make it more explicit:), used for people who want to have a pre release